### DISCUSSION TITLE
ssh stream terminated unexpectedly: rpc error: code

### DISCUSSION THREAD

**Discussion by leosimony** [gravitational/teleport#44640] [Q&A]
Teleport Version: 15.4.6

Installation Method:

1. We are running the teleport as a stack in the Portainer host along with the nginx proxy manager for SSL termination. 
2. A configuration was created on the nginx proxy manager with WebSocket support.
3. I have configured dynamodb for the storage backend and audit logs. 
4. Session recordings are being pushed to the S3 bucket.
5. I can join nodes, create users and connect to the nodes via browser-based, teleport connect and tsh via Windows Powershell.
6. I am not facing any issues while connecting a session to any of the nodes, work on it and disconnect. I am doing this for more than 2 weeks now.

Issues:
I always watch the docker-compose logs to identify if anything is going wrong and the below logs are continuously showing up. 

This particular one I can see if the teleport is being used or not but always showing up in the log.
`2024-07-25T11:09:17Z ERRO             "Error during temporary user cleanup: cant find getent binary\n\texec: \"getent\": executable file not found in $PATH" srv/usermgmt.go:498`

This one appears when there is a session opened by any of the users. 
`2024-07-25T11:13:42Z ERRO [TRANSPORT] ssh stream terminated unexpectedly: rpc error: code = Canceled desc = context canceled pid:7.1 transportv1/transport.go:248`

As I said, I am not facing any issues connecting a session or working on it. I just want to know if this will create a problem in the long run since I am planning to implement it for my team.

Any insights on this is much appreciated.

Regards,
Leo

**Comment by rosstimothy** [comment]
The `ssh stream terminated unexpectedly: rpc error: code = Canceled desc = context canceled pid:7.1 transportv1/transport.go:248` is just log spam that indicates the session has completed. I'll open up a change to silence this to remove any confusion.

**Reply by rosstimothy** [reply]
https://github.com/gravitational/teleport/pull/44660
